uiowa/uiowa
# Update to fix the format of author bio field on older 91st Meridian articles

Resolves #7690 

# How to test

- **_BEFORE_** checking out this branch, run the following:
```
ddev blt ds --site 91stmeridian.iwp.uiowa.edu && ddev drush @iwp91stmeridian.local sqlq "SELECT COUNT(*) FROM node__field_article_author_bio WHERE field_article_author_bio_format = 'minimal'" && ddev drush @iwp91stmeridian.local sqlq "SELECT COUNT(*) FROM node_revision__field_article_author_bio WHERE field_article_author_bio_format = 'minimal'"
```
- It will print out two numbers which will show the number of records that need to be corrected.
- Check out this branch
- Run the above command again. The numbers should be zeros now.
```
ddev drush @iwp91stmeridian.local uli node/1536/edit
```
- Confirm the specific example given in the issue is fixed.
